packages/web-app-files/tests/helpers/path.spec.js

```diff
@@ -0,0 +1,34 @@
+import { getParentPaths } from '../../src/helpers/path'
+
+describe('build an array of parent paths from a provided path', () => {
+  it('should return an empty array on an empty path', () => {
+    const paths = getParentPaths('', false)
+    expect(paths).toHaveLength(0)
+  })
+
+  it('should return an empty array on "/" as path', () => {
+    const paths = getParentPaths('/', false)
+    expect(paths).toHaveLength(0)
+  })
+
+  it('should prepend resulting paths with a "/" if none was given', () => {
+    const paths = getParentPaths('a/b/c', false)
+    expect(paths).toEqual(['/a/b', '/a', ''])
+  })
+
+  it('should make no difference between "a/b/c" and "/a/b/c"', () => {
+    const paths1 = getParentPaths('a/b/c', false)
+    const paths2 = getParentPaths('/a/b/c', false)
+    expect(paths1).toEqual(paths2)
```

Comment:
We could test the `includeCurrent` in a separate test with a `expect(pathWithCurrent).not.toEqual(pathWithoutCurrent)` to make sure it doesn't break silently down the road